packages/fxa-settings/src/pages/Pair/Index/en.ftl

```diff
@@ -0,0 +1,18 @@
+## Pair index page
+
+pair-sync-header = Sync { -brand-firefox } on your phone or tablet
+pair-cad-header = Connect { -brand-firefox } on another device
+pair-already-have-firefox = Already have { -brand-firefox } on a phone or tablet?
+# Clicking this button initiates the pairing process, usually be directing the user to the `about:preferences` page in Firefox
+pair-sync-your-device-button = Sync your device
+# This is immediately preceded by "Sync your device"
+pair-or-download = Or download
+# <linkExternal> is an anchor tag that directs the user to where they can download the { -brand-firefox } app
+pair-scan-to-download = Scan to download { -brand-firefox } for mobile, or send yourself a <linkExternal>download link.</linkExternal>
+# This allows the user to exit the sync/pair flow, and redirects them back to Settings
+pair-not-now-button = Not now
+pair-take-your-data = Take your tabs, bookmarks, and passwords anywhere you use { -brand-firefox }.
```

Comment:
Is this a header or some kind of action like a link or button?

Reply:
Updated with nouns for all of these!